packages/DecorativeIcon/svgs/Oval.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import SVGIcon from '../SVGIcon'
+
+const Oval = props => (
+  <SVGIcon {...props}>
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M21 11C21 14.2712 19.4293 17.1755 17.001 19C15.3295 20.2558 13.2516 21 11 21C5.47715 21 1 16.5228 1 11C1 8.20261 2.14864 5.67349 4 3.85857C5.80375 2.09032 8.27455 1 11 1C16.5228 1 21 5.47715 21 11Z"
+        fill="#F7F7F8"
```

Comment:
There should not be a fill on an icon, the `SVGIcon` will control the colour of the icon.